From 04bdfff007c91fc620ed0b7cd02fd2e1e83bdefc Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri, 12 Nov 2021 18:01:05 +0100
Subject: [PATCH] glib/meson.build: add girdir to glib.2.0.pc

Add girdir to glib-2.0.pc to fix the following build failure with atk:

/home/giuliobenetti/autobuild/run/instance-1/output-1/host/riscv32-buildroot-linux-gnu/sysroot/usr/bin/g-ir-compiler atk/Atk-1.0.gir --output atk/Atk-1.0.typelib --includedir=/usr/share/gir-1.0
Could not find GIR file 'GObject-2.0.gir'; check XDG_DATA_DIRS or use --includedir

Fixes:
 - http://autobuild.buildroot.org/results/2716929db638977e6bf665352a08bd580e1dd0ae

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 glib/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/glib/meson.build b/glib/meson.build
index f78f32969..6c5486699 100644
--- a/glib/meson.build
+++ b/glib/meson.build
@@ -377,6 +377,7 @@ pkg.generate(libglib,
   subdirs : ['glib-2.0'],
   extra_cflags : ['-I${libdir}/glib-2.0/include'] + win32_cflags,
   variables : ['bindir=' + join_paths('${prefix}', get_option('bindir')),
+               'girdir=' + join_paths('${libdir}', '../share/gir-1.0'),
                'glib_genmarshal=' + join_paths('${bindir}', 'glib-genmarshal'),
                'gobject_query=' + join_paths('${bindir}', 'gobject-query'),
                'glib_mkenums=' + join_paths('${bindir}', 'glib-mkenums')],
-- 
2.33.0

